service/settings.go

```diff
@@ -0,0 +1,60 @@
+package service
+
+import (
+	"context"
+
+	"release-manager/pkg/apierrors"
+	"release-manager/service/model"
+
+	"github.com/google/uuid"
+)
+
+type SettingsService struct {
+	authService model.AuthService
+	repository  model.SettingsRepository
+}
+
+func NewSettingsService(authSvc model.AuthService, r model.SettingsRepository) *SettingsService {
+	return &SettingsService{
+		authService: authSvc,
+		repository:  r,
+	}
+}
+
+func (s *SettingsService) Get(ctx context.Context, authUserID uuid.UUID) (model.Settings, error) {
+	if err := s.authService.AuthorizeAdminRole(ctx, authUserID); err != nil {
+		return model.Settings{}, err
+	}
+
+	settings, err := s.repository.Read(ctx)
+	if err != nil {
+		return model.Settings{}, err
+	}
+
+	if err := settings.Validate(); err != nil {
+		return model.Settings{}, apierrors.NewSettingsCorruptedError().Wrap(err)
+	}
```

Comment:
When settings is retrieved from repository, validate function is called in order to check settings validity.